kura/org.eclipse.kura.protocol.can/pom.xml

```diff
@@ -74,7 +54,25 @@
 					</filesets>
 				</configuration>
 			</plugin>
-			
+			<plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Apply formatting also here. Why do you need this shift?

Reply:
When i finish maven compile or install the lib would be deleted. So if shift this the plugins execute clean fist then copy dependencies and the lib dictionary still exist.